app/main.development.js

```diff
@@ -179,6 +181,14 @@ const launchDCRWallet = () => {
     args.push('--testnet');
   }
 
+  // Add any extra args if defined.
+  if (argv.extrawalletargs != undefined) {
+    var extraArgs = argv.extrawalletargs.split(' ');
```

Comment:
e.g. `--extrawalletargs="--appdata=/tmp/path\ with\ spaces"` or some other variation of that with escaping that works :)